package.json

```diff
@@ -17,102 +17,39 @@
     "API Gateway",
     "Velocity"
   ],
-  "author": {
-    "name": "David Hérault",
-    "email": "dherault@gmail.com",
-    "url": "https://github.com/dherault"
-  },
+  "author": "David Hérault <dherault@gmail.com> (https://github.com/dherault)",
   "contributors": [
-    {
-      "name": "Kiryl Yermakou",
-      "email": "rma4ok@gmail.com",
-      "url": "https://github.com/rma4ok"
-    },
-    {
-      "name": "Andrei Popovici",
-      "url": "https://github.com/andreipopovici"
-    },
-    {
-      "name": "DJCrabhat",
-      "url": "https://github.com/djcrabhat"
-    },
-    {
-      "name": "Martin Micunda",
-      "url": "https://github.com/martinmicunda"
-    },
-    {
-      "name": "Andre Rabold",
-      "url": "https://github.com/arabold"
-    },
-    {
-      "name": "John McKim",
-      "url": "https://github.com/johncmckim"
-    },
-    {
-      "name": "Russell Schick",
-      "url": "https://github.com/rschick"
-    },
-    {
-      "name": "Cameron Cooper",
-      "url": "https://github.com/cameroncooper"
-    },
-    {
-      "name": "Piotr Gasiorowski",
-      "url": "https://github.com/WooDzu"
-    },
-    {
-      "name": "Bilal Soylu",
-      "url": "https://github.com/Bilal-S"
-    },
-    {
-      "name": "Michael Staub",
-      "url": "https://github.com/mikestaub"
-    }
+      "Kiryl Yermakou (https://github.com/rma4ok)",
+      "Andrei Popovici (https://github.com/andreipopovici)",
+      "DJCrabhat (https://github.com/djcrabhat)",
+      "Martin Micunda (https://github.com/martinmicunda)",
+      "Andre Rabold (https://github.com/arabold)",
+      "John McKim (https://github.com/johncmckim)",
+      "Russell Schick (https://github.com/rschick)",
+      "Cameron Cooper (https://github.com/cameroncooper)",
+      "Piotr Gasiorowski (https://github.com/WooDzu)",
+      "Norimitsu Yamashita (https://github.com/nori3tsu)",
+      "Bilal Soylu (https://github.com/Bilal-S)",
+      "Michael Staub (https://github.com/mikestaub)"
   ],
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/dherault/serverless-offline/issues"
   },
   "homepage": "https://github.com/dherault/serverless-offline",
-  "dependencies": {    
-    "babel-register": "^6.9.0",
-    "boom": "^3.1.3",
+  "dependencies": {
+    "babel-register": "^6.11.6",
+    "boom": "^3.2.2",
     "coffee-script": "^1.10.0",
-    "hapi": "13.3.0",
+    "hapi": "14.1.0",
     "js-string-escape": "^1.0.1",
     "jsonpath-plus": "^0.15.0",
-    "lodash": "^4.15.0",
-    "velocityjs": "^0.8.0"
+    "lodash.isplainobject": "^4.0.5",
```

Comment:
@mikestaub Thanks for the pull request. I used a few features of "lodash" 4.15.0 and also refactored the remainder of the code to use  main "lodash" library rather then the "lodash.isplainobject" point build. Thus I had removed "lodash.isplainobject" dependency.  If we still use the point build somehow we should add it back, but we will also need to add "lodash" back please.